proxy/src/redis/notifications.rs

```diff
@@ -39,6 +39,27 @@ pub(crate) enum Notification {
     AllowedIpsUpdate {
         allowed_ips_update: AllowedIpsUpdate,
     },
+    #[serde(
+        rename = "/block_public_or_vpc_access_updated",
+        deserialize_with = "deserialize_json_string"
+    )]
+    BlockPublicOrVpcAccessUpdate {
```

Comment:
nit: Should be Updated not Update everywhere, right? Doesn't matter much since serde names the variant, but should still be consistent.

Reply:
I followed the example of `AllowedIpsUpdate` just above. Should I change that one as well then?

Reply:
Ah, sorry, didn't notice. Yes, that should be made consistent too, but not in this PR.

Reply:
Ok I renamed them.